mevislab.github.io/content/tutorials/thirdparty/pytorch/pytorchexample3.md

```diff
@@ -0,0 +1,327 @@
+---
+title: "Example 3: Segment persons in webcam videos"
+date: 2023-05-16
+status: "OK"
+draft: false
+weight: 875
+tags: ["Advanced", "Tutorial", "PyTorch", "Python", "PythonPip", "AI", "Segmentation", "WebCam"]
+menu: 
+  main:
+    identifier: "pytorchexample3"
+    title: "Segment persons in webcam videos."
+    weight: 875
+    parent: "pytorch"
+---
+# Example 3: Segment persons in webcam videos
+
+## Introduction
+This tutorial is based on [Example 2: Face Detection with OpenCV](/tutorials/thirdparty/opencv/thirdpartyexample2 "Example 2: Face Detection with OpenCV"). You can re-use some of the scripts already developed in the other tutorial.
+
+## Steps to do
+Add the macro module developed in previous example to your workspace.
+
+![WebCamTest module](/images/tutorials/thirdparty/pytorch_example3_1.png "WebCamTest module")
+
+Open the internal network of the module via middle mouse button {{< mousebutton "middle" >}} and right click {{< mousebutton "right" >}} on the tab of the workspace showing the internal network. Select *Show Enclosing Folder*.
+
+![Show Enclosing Folder](/images/tutorials/thirdparty/pytorch_example3_2.png "Show Enclosing Folder")
+
+The explorer opens showing the files of your macro module. Copy the *\*.mlab* file somewhere you can remember.
+
+### Create the macro module
+Open the the Project Wizard via {{< menuitem "File" "Run Project Wizard">}} and select *Macro Module*. Click *Run Wizard*.showing the internal network.
+
+![Project Wizard](/images/tutorials/thirdparty/pytorch_example3_3.png "Project Wizard")
+
+Define module properties as seen below, you can chose any name you like. Click *Next*.
```

Comment:
Define the module properties as shown below, though you can chose your own name.